api/v1alpha1/istio_types.go

```diff
@@ -35,6 +42,10 @@ type IstioSpec struct {
 	// +kubebuilder:validation:Enum=v1.20.1;v1.20.0;v1.19.5;latest
 	Version string `json:"version"`
 
+	// Defines the update strategy to use when the version in the Istio CR is updated.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Update Strategy"
+	UpdateStrategy *IstioUpdateStrategy `json:"updateStrategy,omitempty"`
```

Comment:
if omitted, can we still see the appliedValues somewhere?

Reply:
Not sure what appliedValues have to do with this. If you omit `updateStrategy`, it currently defaults to `InPlace` (later we'll probably change the default to `RevisionBased`). In either case, the applied values are visible in the underlying IstioRevision object.

Reply:
sorry, not appliedvalues.... my question is: will it be possible to inspect what (default) value was applied for any omitted field?

Reply:
Maybe. But we have to add a mutatingwebhook, which we currently don't have yet.